### PR TITLE
Step By Step Instructions Review

### PR DESCRIPTION
**Pull request by shannonxtreme**
This is a 🔦 documentation change.

## Summary

Reworking the step-by-step documentation for clarity, flow, and accuracy.

### CODE REVIEW

**Comment by shannonxtreme** [comment]
@DirtyF draft request for changes to tutorials section. I'll undraft when all topics are ready for review.

**Review by DirtyF — APPROVED** [review]
Awesome, thanks for the review 🙏

File: docs/_docs/ruby-101.md [hunk 1]
```diff
@@ -3,24 +3,27 @@ title: Ruby 101
...
+
+Jekyll is a gem. Many Jekyll [plugins]({{ '/docs/plugins/' | relative_url }}) are also gems, including
 [jekyll-feed](https://github.com/jekyll/jekyll-feed),
 [jekyll-seo-tag](https://github.com/jekyll/jekyll-seo-tag) and
 [jekyll-archives](https://github.com/jekyll/jekyll-archives).
 
 ## Gemfile
 
-A `Gemfile` is a list of gems required for your site. For a simple Jekyll site it might look something like this:
+A `Gemfile` is a list of gems used by your site. Every Jekyll site has a Gemfile in the main folder.
```
Comment (antonvasilev52) [inline docs/_docs/ruby-101.md]: In this chapter, does the use of different formatting (monospace "`Gemfile`" vs plain text "Gemfile") implies different meanings of this word?

**Comment by shannonxtreme** [comment]
Nope. The monospace gemfile *should* be used when referring directly to the
file in the file system. The regular font should be used to refer
inderectly to the gemfile.

For example:

"Open the `Gemfile` in Notepad."

"The Gemfile is used to define parameters for blah"

**Comment by nnmuniz** [comment]
Gemfile é usado para definir parâmetros

**Comment by nnmuniz** [comment]
Gemfile is used to define parameters

**Comment by DirtyF** [comment]
Let's publish this.

@jekyll: merge +docs

**Comment by luismedicina23** [comment]
@shannonxtreme 

Hello im newbee gere
And im dona speek english só much
Im brazilian 

Sry

**Review by luismedicina23 — COMMENTED** [review]
@shannonxtreme @DirtyF @luismedicina23 copy youeself
But um begginer And brazi
Im trying swear

Im sry

**Pull request merged**